File: apphub-integration/src/main/java/com/github/saphyra/apphub/integration/framework/endpoints/skyxplore/SkyXploreAdminEndpoints.java
package com.github.saphyra.apphub.integration.framework.endpoints.skyxplore;

public class SkyXploreAdminEndpoints {
    public static final String SKYXPLORE_GAME_ADMIN_MAIN_PAGE = "/web/skyxplore/game/admin";
    public static final String SKYXPLORE_GAME_ADMIN_LIST_PAGE = "/web/skyxplore/game/admin/{type}/{gameId}";
    public static final String SKYXPLORE_GAME_ADMIN_DETAILS_PAGE = "/web/skyxplore/game/admin/{gameId}/{type}/{itemId}";

    public static final String SKYXPLORE_GAME_ADMIN_GET_BY_TYPE = "/api/skyxplore/game/admin/{type}";
    public static final String SKYXPLORE_GAME_ADMIN_GET_ITEM = "/api/skyxplore/game/admin/{gameId}/{type}/{itemId}";
}

File: apphub-frontend/src/modules/feature/skyxplore/admin/SkyXploreAdminEndpoints.js
import { Endpoint, RequestMethod } from "common/js/dao/dao";

export const SKYXPLORE_ADMIN_MAIN_PAGE = "/web/skyxplore/game/admin";
export const SKYXPLORE_ADMIN_DETAILS_PAGE = new Endpoint(RequestMethod.GET, "/web/skyxplore/game/admin/{gameId}/{type}/{id}");

export const SKYXPLORE_GAME_ADMIN_GET_BY_TYPE = new Endpoint(RequestMethod.GET, "/api/skyxplore/game/admin/{type}");
export const SKYXPLORE_GAME_ADMIN_GET_ITEM = new Endpoint(RequestMethod.GET, "/api/skyxplore/game/admin/{gameId}/{type}/{itemId}");
